### PR TITLE
core: add InfoConfirm

### LINKED ISSUE #512: Add Info button to Recovery homescreen for Super Shamir
In Recovery Homescreen in case we already know it is a Super Shamir, let's display an additional "Info" button, which will show, which groups and shares are needed.

Unfortunately we are using the Confirm dialog, which works with two buttons only, so this will need some refactor.

Fast prototype:

emu00000000

### PR DESCRIPTION
**Pull request by tsusanka**
This adds another Confirm dialog with an Info button and closes #512.

Although the Confirm dialogs bloat a little I think we might use the Info button somwhere else as well. Or we refactor this as soon as we have a new styleguide.

### CODE REVIEW

**Comment by tsusanka** [comment]
The tests fail for the same reason they fail on iss442-supershamir-refactor

**Comment by tsusanka** [comment]
@ciny do you think you could add a test for the info screen as well? just to click through it

**Comment by ciny** [comment]
I removed the function, the tests pass now. However adding the info screen test will require modifying the debuglink. I think that is out of scope of this PR.

File: core/src/apps/common/confirm.py [hunk 1]
```diff
@@ -47,6 +47,37 @@
...
+        content, confirm, confirm_style, cancel, cancel_style, info, info_style
+    )
+
+    while True:
+        if __debug__:
+            result = await ctx.wait(dialog, confirm_signal())
+        else:
+            result = await ctx.wait(dialog)
+
+        if result == INFO:
```
Comment (matejcik) [inline core/src/apps/common/confirm.py]: this must be `if result is INFO` otherwise this is always false

Comment (matejcik) [reply core/src/apps/common/confirm.py]: hmm, no, the result is true. but still this should be `result is INFO` to clarify intent

Comment (tsusanka) [reply core/src/apps/common/confirm.py]: The `result` returns one of 

```
CONFIRMED = object()
CANCELLED = object()
INFO = object()
```

These are defined as unrelated standalone new objects. How is `is` any better?

Comment (ciny) [reply core/src/apps/common/confirm.py]: done

Comment (matejcik) [reply core/src/apps/common/confirm.py]: `is` is explicit identity testing, which is what we want here.
`==` also does identity testing in this case, but not necessarily in general or if we change the definition of INFO/CONFIRMED/CANCELLED

**Pull request merged**